geoportal/webpack.api.js.mako

```diff
@@ -0,0 +1,71 @@
+const path = require('path');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+
+const destDir = path.resolve(__dirname, '${package}_geoportal/static-ngeo/build/');
+
+const babelPresets = [[require.resolve('@babel/preset-env'), {
+  targets: {
+    browsers: ['last 2 versions', 'Firefox ESR', 'ie 11'],
+  },
+  modules: false,
+  loose: true
+}]];
+
+module.exports = (env, argv) => {
+  const library = argv.library ? argv.library : '${package}';
+  return {
+    entry: path.resolve(__dirname, '${package}_geoportal/static-ngeo/api/index.js'),
+    devtool: 'source-map',
+    module: {
+      rules: [{
+        test: /\.js$/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: babelPresets,
+            babelrc: false,
+            comments: false,
+            cacheDirectory: '/build/bable-loader-cache/',
+            plugins: [
+              require.resolve('@babel/plugin-syntax-object-rest-spread'),
+              require.resolve('@babel/plugin-transform-spread'),
+              require.resolve('@camptocamp/babel-plugin-angularjs-annotate'),
+            ]
+          }
+        }
+      }]
+    },
+    output: {
+      filename: 'api.js',
+      path: destDir,
+      libraryTarget: 'umd',
+      globalObject: 'this',
+      libraryExport: 'default',
+      library: library
+    },
+    optimization: {
+      minimizer: [
+        new UglifyJsPlugin({
+          parallel: true,
+          sourceMap: true,
+          uglifyOptions: {
+            compress: false
+          }
+        })
+      ]
+    },
+    resolve: {
+      modules: [
+        '/usr/lib/node_modules',
+        '/usr/lib/node_modules/ol/node_modules',
+        '/usr/lib/node_modules/proj4/node_modules',
+      ],
+      alias: {
+        api: '/usr/lib/node_modules/ngeo/api/src',
+      }
+    },
+    resolveLoader: {
+      modules: ['/usr/lib/node_modules'],
+    }
+  };
+};
```

Comment:
Can be not interpreted by using `{{package}}` in the scaffold :-)

Reply:
done